### ISSUE TITLE
About Train CycleGAN

### ISSUE THREAD

**Issue by Daipuwei** [kevinhkhsu/DA_detection#13]
The image size of Cityscape and Foggy Cityscape dataset is 1024 * 2048. So when training CycleGAN, the input of network is also 1024 * 2048? I have a Tesla P100,which memory is 16G

**Comment by kevinhkhsu** [comment]
You can train the network with half the resolution to fit into the memory.

**Comment by Daipuwei** [comment]
> You can train the network with half the resolution to fit into the memory.

could you tell me your CycleGAN training details in Cityscape and Foggy Cityscape dataset? For example the learning rate, batchsize,optimizer, data augmentation and so on?

**Comment by kevinhkhsu** [comment]
I followed the default settings used in [PyTorch-CycleGAN](https://github.com/aitorzip/PyTorch-CycleGAN).
